habitat-lab/habitat/config/benchmark/rearrange/cat_nav_to_rec.yaml

```diff
@@ -27,27 +28,28 @@ habitat:
       - data/objects/ai2thorhab/configs/objects/
       - data/objects/floorplanner/configs/objects/
     debug_render_goal: False
+    debug_render: False
     concur_render: True
     auto_sleep: True
+    requires_textures: False
+    kinematic_mode: True
     agents:
       main_agent:
         radius: 0.3
         height: 1.41
         robot_urdf: data/robots/hab_stretch/urdf/hab_stretch.urdf
         robot_type: "StretchRobot"
         sim_sensors:
-           head_rgb_sensor:
-             height: 640
-             width: 480
-             hfov: 42
            head_depth_sensor:
-             height: 640
-             width: 480
+             height: 160
```

Comment:
I havent. Its going to be much slower to train that model and I dont think it would be too much better because we are only using depth and segmentation. 

Also, you have been using 256x256 but were downscaling to 128x128 before passing it to the policy. I am not downsampling so essentially the image size hasn't reduced a lot.